README.md

```diff
@@ -18,14 +18,6 @@ This encoder has been optimized to achieve excellent performance levels using 13
 
 - Real-time encoding of up to six 4Kp60/10-bit streams on the Platinum 8180 with M12 in the visual quality mode
 
-SVT-HEVC Encoder also supports 3 modes:
```

Comment:
the read me file might need more review, as there's still a mention of M12 and visual quality mode

Reply:
Let's redo the benchmark, include those in the tuning guide.